src/main/scala/com/github/simplesteph/ksm/source/BitbucketServerSourceAcl.scala

```diff
@@ -58,6 +62,10 @@ class BitbucketServerSourceAcl extends SourceAcl {
     // super important in order to properly fail in case a timeout happens for example
     request.enableThrowingIOException(true)
 
+    if (branch.isDefined) {
```

Comment:
I think the scala way is something like `branch.forEach { b =>  request.queryParam("until", b) }` but please check too

Reply:
Thanks. Done